consensus/istanbul/backend/engine_test.go

```diff
@@ -75,7 +75,7 @@ func makeCommittedSeals(hash common.Hash) [][]byte {
 func newBlockChain(n int, items ...interface{}) (*blockchain.BlockChain, *backend) {
 	// generate a genesis block
 	genesis := blockchain.DefaultGenesisBlock()
-	genesis.Config = params.TestChainConfig
+	genesis.Config = &params.ChainConfig{ChainID: big.NewInt(1)}
```

Comment:
I changed to allocate new chainConfig because TestChainConfig can be accessed by multiple tests.

Reply:
Why can't this `params.TestChainConfig`  be shared?

Reply:
I revised `Test_GossipSubPeerTargets` test function to use newBlockChain.
```
	_, backend := newBlockChain(1, istanbulCompatibleBlock(big.NewInt(0)))
```
After the revision, `TestSnapshot` is failed. I thought this is because TestChainConfig is modified by newBlockChain. By allocating new chainConfig, `TestSnapshot` became not failed.